hudi-common/src/main/java/org/apache/hudi/common/table/log/BaseHoodieLogRecordReader.java

```diff
@@ -410,6 +418,34 @@ private void scanInternalV1(Option<KeySpec> keySpecOpt) {
     }
   }
 
+  private static void prepareRecordPositions(String blockPositionStr,
+                                             List<Integer> blockPositions,
+                                             Set<Integer> globalPositions,
+                                             boolean positionBasedMergeEnabled,
+                                             boolean useEventOrdering) throws IOException {
```

Comment:
Yeah, i have checked Ethan's PR, and tried to use the precombine_field_name config to define this variable. That is, when precombine_field_name exists, we use event time ordering, otherwise, processing time ordering. let me know if this works.